fusionfusion/annotationFunction.py

```diff
@@ -222,23 +226,23 @@ def merge_fusion_result(input_dir, output_file_path):
         with open(input_dir + "/ms2.fusion.result.txt") as hIN:
             for line in hIN:
                 F = line.rstrip('\n').split('\t')
-                key = '\t'.join(F[0:7] + F[8:12])
+                key = '\t'.join(F[0:7] + F[8:])
```

Comment:
Is it the same as [#2](https://github.com/chrovis-genomon/fusionfusion/pull/2#discussion_r539763514)?

Isn't  [no-blat](https://github.com/chrovis-genomon/fusionfusion/pull/3/commits/2be07313731c234a804343f6fc66e2cb9c7f7366)'s merge source old?

Reply:
My bad! The base branch should have been `feature/no-blat`, not `feature/ecr`.

Reply:
Fixed. Thanks for pointing it out!